lib/helpers.py

```diff
@@ -10,437 +10,598 @@
 
 cache_folder = "cache-v{0}".format(__version__)
```

Comment:
Maybe capitalize, since it's module-level and "kind of" constant?

Reply:
No, `__version__` is a Python way to specify version: https://www.python.org/dev/peps/pep-0396/

Reply:
Agree, technically it's not a constant (even though doesn't change during execution) -> shouldn't be capitalized.